### ISSUE TITLE
Feature request: get the trim coordinates

### ISSUE THREAD

**Issue by yjutard** [duniul/trim-image-data#4]
Hi,

this package looks promising and could solve my trim issue! Thanks for sharing!

I need to trim SVGs. The plan is to convert them to PNG in their native viewport, then extract the coordinate of the content then trim them in SVG with a group shift + viewport adjustment.

To do that, I'd need this package to not do the crop but instead to give me the coordinates of the content (or the crop params):

```
import trimImageData from 'trim-image-data';

const trimmedContentCoordinates = trimImageData.getCoordinates(imageData, {...});
```

It could return either the content coordinates or crop options `top, right, bottom, left`

What do you think?
Thanks!

**Comment by duniul** [comment]
Hey!

Breaking out and exporting e.g. a `getTrimEdges()` function should be easy, I'll look into adding it a little later today 👍

**Comment by duniul** [comment]
@yjutard please have a look at #5, I think it should cover your needs!

Went with `getTrimEdges()` instead of `getCoordinates()` as it returns the number of pixels to crop/trim from each side (which is what it uses internally to pass to `crop-image-data`). The coordinates can easily be calculated from the result if you have the image data available though!

You can use it like so:

```ts
import { getTrimEdges } from 'trim-image-data';

const { top, right, bottom, left } = getTrimEdges(imageData, {...});
const topLeftCoordinate = { x: left, y: top };
const bottomRightCoordinate = { x: imageData.width - right, y: imageData.height - bottom };
```

Let me know if you think it makes sense and I'll merge and release it asap ✅

**Comment by yjutard** [comment]
That looks great!
Thanks a lot!

**Comment by yjutard** [comment]
I'm in node.js, I may have to convert to ImageData...

**Comment by duniul** [comment]
@yjutard released in [v2.1.0](https://github.com/duniul/trim-image-data/releases/tag/v2.1.0) ✅

**Comment by yjutard-while-at-atlassian** [comment]
Thanks a lot for your help!
